lambda_function.py

```diff
@@ -202,10 +210,12 @@ def create_stack_set_flow(self) -> None:
 
         try:
             self.create_stack_set()
+            logger.error(f"StackSet '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}' created")
         except Exception as e:
-            if "NameAlreadyExistsException" in repr(e):
+            repr_e_lower = repr(e).lower()
+            if "already" in repr_e_lower and "exists" in repr_e_lower:
                 logger.error(
-                    f"Received NameAlreadyExistsException for stack_set '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}'. Error: {repr(e)}")
+                    f"StackSet '{self.MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME}' already exists. Skipping this step.")
```

Comment:
maybe we should make in warning rather than error 
it is confusing in the CW logs

Reply:
done